script/get-images-versions.sh

```
@@ -0,0 +1,147 @@
+        tail -1)
+
+    if echo "$tags" | grep -qE "^v${latest}$"; then
+        latest="v${latest}"
+    fi
+
+    echo "$latest"
+}
+
+# Fetch all tags in parallel
+tmpdir=$(mktemp -d)
+trap "rm -rf $tmpdir" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)